starlark/unpack.go

```diff
@@ -44,23 +44,23 @@ type Unpacker interface {
 //
 // Examples:
 //
-//      var (
-//          a Value
-//          b = MakeInt(42)
-//          c Value = starlark.None
-//      )
+//	var (
```

Comment:
fyi -- these indentation changes come from go fmt in Go 1.22. I can separate them out into a separate pr if that's easier.

Reply:
Don't worry, it's fine.

Reply:
While you're here, would you mind fixing these preexisting things?

L28: s/integer types uses/integer types use/
L36:  s/where or not/whether or not/

s/interface{}/any/g throughout this function

thanks